NewForce-Cohort-4/nutshell-team-odd-eclipse
# Pp article form

# Description
Created a form for new articles to be created in, form prints to the dom when you click create article button, created save and delete functions that work with corresponding buttons, article list prints a refreshed list after using buttons and the list prints in reference to the timestamp being used when the save button is clicked
Fixes # (issue)

Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
log in as a user, make sure only the articles that are assigned to your users Id are showing, click create article, save the article, then see the new article appear at the top of the list, delete an article that you don't want anymore
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
